crates/full-node/sov-sequencer/src/preferred/transaction_subscriptions.rs

```diff
@@ -254,6 +254,7 @@ impl<S: Spec, Rt: Runtime<S>> TransactionCache<S, Rt> {
         Ok(Some(AcceptedTx {
             tx: FullyBakedTx {
                 data: tx.body.unwrap_or_default(),
+                sequencing_data: None, // TODO
```

Comment:
What about this? Is for later, if yes then let's make an issue for it and put it in the comment ()unless it will be done in a followup